launchable/commands/subset.py

```diff
@@ -247,4 +254,28 @@ def run(self):
 
                 self.print(output)
 
+            build_name, test_session_id = parse_session(session_id)
+            org, workspace = get_org_workspace()
+
+            header = ["", "Candidates",
+                      "Estimated duration (%)", "Estimated duration (min)"]
+            rows = [
+                ["Subset", len(output), summary["subset"]["rate"],
+                 summary["subset"]["duration"]],
+                ["Remainder", len(rests), summary["rest"]
+                 ["rate"], summary["rest"]["duration"]],
+                [],
+                ["Total", len(output) + len(rests), summary["subset"]["rate"] + summary["rest"]
+                 ["rate"], summary["subset"]["duration"] + summary["rest"]["duration"]],
+            ]
+
+            if is_brainless:
+                click.echo(
+                    "INFO: Your model is currently training", err=True)
```

Comment:
"Your model is currently training" sounds a bit off to me. I believe "Your model is currently in training." more suitable to your intention!

Reply:
Thank you for your advice! I'll fix it

Reply:
Only for attention, no important mean.

Reply:
fixed
https://github.com/launchableinc/cli/commit/ad1edf0b64bf8193224a4f399cd2fd8690d9cdfc